test/aria/templates/css/imgprefix/ImgPrefixTemplateTestCase.js

```diff
@@ -75,12 +79,42 @@ Aria.classDefinition({
                 var tmp = this._cleanUrls(urls[i]);
                 this.assertTrue(tmp === this.imgPaths[i], "The framework is adding a prefix for image urls inside CSS templates, it shouldn't.");
             }
+            this._testImageIsActuallyLoadedAfterPrefixing();
+        },
+
+        _testImageIsActuallyLoadedAfterPrefixing : function () {
+            aria.core.AppEnvironment.setEnvironment({
+                "imgUrlMapping" : function (url) {
+                    return aria.core.DownloadMgr.resolveURL("aria/css/atskin/imgs/" + url);
+                }
+            }, {
+                fn : this._loadThirdTpl,
+                scope : this
+            }, true);
+        },
+
+        _loadThirdTpl : function () {
+            this._replaceTestTemplate({
+                template : "test.aria.templates.css.imgprefix.TemplateWithRealImage"
+            }, this._afterThirdTemplateLoaded);
+        },
+
+        _afterThirdTemplateLoaded : function () {
+            var container = Aria.$window.document.getElementById("container-real-image");
+            var style = aria.utils.Dom.getStyle(container, "background-image");
```

Comment:
`aria.utils.Dom.getStyle` is expecting the property in camel case:

```
var style = aria.utils.Dom.getStyle(container, "backgroundImage");
```

Otherwise, the test fails in Firefox 3 and Firefox 11...